### ISSUE TITLE
Breaks Visual Studio Rebuild when RazorCompileOnBuild is set to true

### ISSUE THREAD

**Issue by Nikkelmann** [patrickdemooij9/SeoToolkit.Umbraco#97]
When the project setting RazorCompileOnBuild is set to true the project fails to rebuild in Visual Studio.

```
Rebuild started...
1>------ Rebuild All started: Project: MyUmbracoProject, Configuration: Debug Any CPU ------
Restored C:\udv\repos\MyUmbracoProject\MyUmbracoProject.csproj (in 761 ms).
1>Clear old SeoToolkit data
1>Clear old SeoToolkit views
1>CSC : warning CS8785: Generator 'RazorSourceGenerator' failed to generate source. It will not contribute to the output and compilation errors may occur as a result. Exception was of type 'NotSupportedException' with message 'This API should not be invoked. We should instead be relying on the RazorSourceDocument associated with this item instead.'
1>CSC : error CS2001: Source file 'C:\udv\repos\MyUmbracoProject\Views\ScriptManager\GoogleAnalytics\Script.cshtml' could not be found.
1>CSC : error CS2001: Source file 'C:\udv\repos\MyUmbracoProject\Views\ScriptManager\GTM\BodyTop.cshtml' could not be found.
1>CSC : error CS2001: Source file 'C:\udv\repos\MyUmbracoProject\Views\ScriptManager\GTM\HeadBottom.cshtml' could not be found.
1>CSC : error CS2001: Source file 'C:\udv\repos\MyUmbracoProject\Views\ScriptManager\Hotjar\Script.cshtml' could not be found.
1>Done building project "MyUmbracoProject.csproj" -- FAILED.
========== Rebuild All: 0 succeeded, 1 failed, 0 skipped ==========
```

Manually calling Clear and then Build works fine:
```
Clean started...
1>------ Clean started: Project: MyUmbracoProject, Configuration: Debug Any CPU ------
1>Clear old SeoToolkit data
1>Clear old SeoToolkit views
========== Clean: 1 succeeded, 0 failed, 0 skipped ==========
```
```
Build started...
Restored C:\udv\repos\MyUmbracoProject\MyUmbracoProject.csproj (in 472 ms).
1>------ Build started: Project: MyUmbracoProject, Configuration: Debug Any CPU ------
1>MyUmbracoProject -> C:\udv\repos\MyUmbracoProject\bin\Debug\net6.0\MyUmbracoProject.dll
1>Copying Umbraco json schema file
1>Copying SeoToolkit files: C:\Users\user\.nuget\packages\seotoolkit.umbraco\2.0.1\build\..\App_Plugins\SeoToolkit\**\*.* - #53 files
1>Copying SeoToolkit.ScriptManager views: C:\Users\user\.nuget\packages\seotoolkit.umbraco\2.0.1\build\..\Views\ScriptManager\**\*.* - #4 files
========== Build: 1 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
```

**How to recreate**
1) Create new umbraco project (10+)
2) Add package SeoToolkit.Umbraco, Version 2.0.1
3) Set RazorCompileOnBuild=true in csproj
4) Rebuild from Visual Studio

**Comment by patrickdemooij9** [comment]
That's strange. Will try to reproduce and fix when I get back from vacation. Thanks for the report!

**Comment by patrickdemooij9** [comment]
Hi @Nikkelmann 
This should now be fixed in 2.1.0 along with the change to move everything to static assets!

Thanks for reporting the issue. I hadn't even thought that such configuration would give issues
